### PR TITLE
 #165175671 Removes duplicated swagger endpoints 

### PR DESCRIPTION
**Pull request by anyatibrian**
#### What does this PR do?
Fixes  a bug of duplicated endpoints on swagger
#### Description of Task to be completed?
- refactor the code to contain two serializers
- refactor the views to have two separate view class
- refactor the email message 

#### How should this be manually tested?
After cloning the repo, cd into it and run the following commands to activate the virtual environment
- `$ virtualenv venv`
- `$ source venv/bin/activate`
Install all requirements
- `$ pip install -r requirements.txt`
Make migrations
- `$ python manage.py makemigrations`
- `$ python manage.py migrate`
Run the server 
- `$ python manage.py runserver`
#### What are the relevant pivotal tracker stories?
[#165175671](https://www.pivotaltracker.com/story/show/165175671)
#### Screen Shots
when user enters a valid email address he/she get message prompting them to check their email
Screenshot 2019-04-01 at 19 08 52
upon which a password reset link is sent 
Screenshot 2019-04-09 at 16 02 09
 following the link they will reach an endpoint where the are to set their newpassword 
Screenshot 2019-04-01 at 19 10 59

### CODE REVIEW

File: requirements.txt [hunk 1]
```diff
@@ -46,13 +48,15 @@ pyasn1-modules==0.2.4
 pycodestyle==2.5.0
 pycparser==2.19
 pyflakes==2.1.1
-pyjwt==1.7.1
+PyJWT==1.7.1
 pylint==2.3.1
+python-decouple==3.1
```
Comment (e-ian) [inline requirements.txt]: @anyatibrian, why do we have the python-decouple dependency here again. We are not using it anymore. Kindly remove.

**Comment by e-ian** [comment]
@anyatibrian Our coverage/coveralls check is complaining. Add some tests.

**Review by patrickf949 — APPROVED** [review]
LGTM

**Review by e-ian — APPROVED** [review]
LGTM

**Review by inno-asiimwe — APPROVED** [review]
LGTM

**Pull request merged**